tools/osx-pkg-postinstall.sh

```diff
@@ -3,3 +3,4 @@
 # TODO Can we extract $PREFIX from the installer?
 cd /usr/local/bin
 ln -sf ../lib/node_modules/npm/bin/npm-cli.js npm
+ln -sf ../lib/node_modules/npm/bin/npx-cli.js npx
```

Comment:
that is correct... that script does nothing sadly... although I agree that we should either be explicit or just remove it